BasicJSONTableView/CardStore.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  CardStore.swift
+//  BasicJSONTableView
+//
+//  Created by Nick Nguyen on 10/8/20.
+//  Copyright © 2020 Declan McKenna. All rights reserved.
+//
+
+import Foundation
+
+struct Card {
+    let code: String
+    let image: URL
+    let suit: String
+    
+    enum CodingKeys: String, CodingKey {
+        case code
+        case image
+        case suit
+    }
+}
+
+extension Card: Decodable {
+    init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
```

Comment:
I think the synthesized `CodingKeys` and `init(from decoder)` will work here. It's been a while since I worked on this so I could be wrong although I think you can get by without lines 16-34 as the variables names match up and surprisingly the synthesized `init` can handle `URL`s.